### ISSUE TITLE
Feature request: Allow interface to add 3rd file to 2 file compare

### ISSUE THREAD

**Issue by dmz73vu** [WinMerge/winmerge#464]
When comparing 2 files it would be useful to add 3rd file to compare/merge.
This often happens when comparing two revisions of the same file in a VCS and then need to merge just some changes to working copy of a file in unrelated branch.

**Comment by 4evermaat** [comment]
I agree.  Wondering why the user must start over (select all 3 files all over again from scratch) in order to compare 3 files

**Comment by Nagidrop** [comment]
Wait it's still not possible? I searched for an option/procedure to add 3rd file to 2-file comparison and found out this GitHub issue. This feature would help a lot in many cases where you changed 1 of the 2 files in comparison, make a new copy from that and need to check how it's different from the originals.

**Comment by Darskiy** [comment]
Still actual
